samples/openthread/cli/README.rst

```diff
@@ -339,6 +339,85 @@ To test the Thread 1.2 features, complete the following steps:
    .. note::
         The DUA registration with the Backbone Router is not yet supported.
 
+#. On the leader kit, list IPv6 addresses and pick the Link-Local address for the Link Metrics commands on the second kit
+
+   .. code-block:: console
+
+      uart:~$ ot ipaddr
+      fd00:7d03:7d03:7d03:84c9:572d:be24:cbe
+      fdde:ad00:beef:0:0:ff:fe00:fc10
+      fdde:ad00:beef:0:0:ff:fe00:fc38
+      fdde:ad00:beef:0:0:ff:fe00:fc00
+      fdde:ad00:beef:0:0:ff:fe00:7000
+      fdde:ad00:beef:0:a318:bf4f:b9c6:5f7d
+      fe80:0:0:0:10b1:93ea:c0ee:eeb7
+
+#. On the router kit:
+
+   * Change to SED mode
```

Comment:
This is not allowed once the device is a Router. I should be configured as SED from the beginning.